server/stats.go

```diff
@@ -80,14 +82,22 @@ func (s *Server) GetStats(ctx context.Context, in *pb.GetStatsRequest) (
 			result[dcid] = tmp[dcid].(*pb.ObsTimeSeries)
 		}
 	}
-
 	// Read data from base cache if no branch cache data is found.
```

Comment:
Update the comment to reflect current code... i.e., clarify that for DCIDs missing stats in branch cache, we'll get from base cache.

Reply:
Done